nba_py/__init__.py

```diff
@@ -1,16 +1,30 @@
 from requests import get
-from datetime import datetime
+from datetime import datetime, timedelta
 from nba_py.constants import League
+import os
 
 HAS_PANDAS = True
 try:
     from pandas import DataFrame
 except ImportError:
     HAS_PANDAS = False
 
+HAS_REQUESTS_CACHE = True
```

Comment:
@grillazz Unless I am misunderstanding what is going on, I believe you would just have to merge upstream master into your local branch (via `git fetch upstream` and `git merge upstream/master` see [syncing a fork](https://help.github.com/articles/syncing-a-fork/)) and then push to this branch and this PR would update itself.